src/AppInstallerCLICore/Commands/InstallCommand.cpp

```diff
@@ -122,6 +122,11 @@ namespace AppInstaller::CLI
 
     void InstallCommand::ExecuteInternal(Context& context) const
     {
+        if (context.Args.GetArg(Execution::Args::Type::Query).empty())

```

Comment:
Unfortunately this will block usages like "winget install --id Microsoft.Terminal"

We might need a more sophisticated logic to detect "there's nothing in the search request".

Also I think this should apply to uninstall command as well, in this case, we may want to move the logic to base command.